README.md

```diff
@@ -1,7 +1,49 @@
-# party-radar-app
+# event-radar-app
 Holds all the code for the Android app
+TODO: Add demo
+
+## Overview of functionalities
+
+Event Radar is an innovative app designed to effortlessly facilitate connections among individuals by streamlining event discovery, registration, and networking in one place. It allows users to discover and register for upcoming events in their local area, enabling them to connect seamlessly with friends and new acquaintances they meet during the event. Event hosts can easily create and manage events while leveraging the app's unique QR code system to scan tickets and easily interact with event attendees.
+
+### Features
+
+- **Event Discovery:** Effortlessly find events based on your interests and location and view their position on the map.
+
+- **Easy Registration:** Register for events with just a few taps.
+
+- **Networking:** Connect with new friends and save their contact information by scanning QR codes.
+
+- **Messaging:** Continue the conversation with new acquaintances after the event.
+
+- **Event Hosting:** Create and manage events with robust tools for ticketing and attendee tracking.
+
+- **Quick verification:** Scan attendees' QR codes to confirm their presence on the event's participant list.
+
+
+### Testing
+
+The code is thoroughly tested, achieving 88% line coverage across our features with several hundred tests. Each screen displayed to users is covered by an Android UI test, ensuring the interface works correctly. Additionally, numerous unit tests verify the functionality of individual components. We also conduct end-to-end tests to ensure a smooth and accurate user flow, providing an optimal user experience.
```

Comment:
The 88% might change - should update it later. Alternatively, say above 80% coverage.